### PR TITLE
chore: Remove unnecessary error codes

### PR DESCRIPTION
**Pull request by sfoslund**
#### Details

Follow up on PR comment: https://github.com/microsoft/axe-windows/pull/759#issuecomment-1275385362, removing error codes that are no longer relevant

##### Motivation

Clean up code

#### Pull request checklist

- [n/a] Addresses an existing issue: #0000

### CODE REVIEW

File: src/Automation/Resources/ErrorMessages.resx [hunk 1]
```diff
@@ -121,22 +121,21 @@
     <value>The directory "{0}" was invalid.</value>
   </data>
   <data name="ErrorFailToGetRootElementsOfProcess" xml:space="preserve">
-    <value>ERROR Automation017: Failed to get the root element(s) of the specified process({0}) : {1}</value>
```
Comment (DaveTryon) [inline src/Automation/Resources/ErrorMessages.resx:124]: AutomationIntegrationTests checks for this error in lines 248 and 264. Looking at the code, maybe we just always Assert.Inconclusive if an error is thrown while scanning ?

File: src/AutomationTests/AutomationIntegrationTests.cs [hunk 2]
```diff
@@ -243,9 +243,9 @@ private IReadOnlyCollection<WindowScanOutput> ScanSyncWithProvisionForBuildAgent
             {
                 return scanner.Scan(null).WindowScanOutputs;
             }
-            catch (AxeWindowsAutomationException e)
+            catch (AxeWindowsAutomationException)
```
Comment (DaveTryon) [inline src/AutomationTests/AutomationIntegrationTests.cs:246]: Maybe this should be all Exception types?
```suggestion
            catch (Exception)
```

File: src/AutomationTests/AutomationIntegrationTests.cs [hunk 3]
```diff
@@ -259,9 +259,9 @@ private IReadOnlyCollection<WindowScanOutput> ScanAsyncWithProvisionForBuildAgen
             {
                 return scanner.ScanAsync(null, CancellationToken.None).Result.WindowScanOutputs;
             }
-            catch (AxeWindowsAutomationException e)
+            catch (AxeWindowsAutomationException)
```
Comment (DaveTryon) [inline src/AutomationTests/AutomationIntegrationTests.cs:262]: Maybe this should be all Exception types?
```suggestion
            catch (Exception)
```

**Review by DaveTryon — APPROVED** [review]

**Comment by sfoslund** [comment]
Amending last commit to retrigger checks, as license check was stuck. No real changes.

**Pull request merged**